keps/sig-testing/3041-node-conformance-and-features/README.md

```diff
@@ -0,0 +1,782 @@
+and start relying on alternative labels for filtering in environments.
+
+- Rename NodeFeature to Feature. https://testgrid.k8s.io/sig-node-containerd#node-e2e-features will execute tests for all Features, excluding everything that needs a special environment, see labels above.
+- Document the meaning of NodeConformance. NodeConformance will indicate ALL tests that are testing enabled out of the box functionality that is not runtime or k8s distro specific. These tests may still require an additional environment set up. NodeConformance tests may have all labels specifying its environment requirements, like Slow, Disruptive, Special, etc.
+- Introduce a new periodic job that runs all NodeConformance tests with all FeatureGates turned off.
+
+### NodeConformance
+
+NodeConformance label represents tests that have to be working on all environments
+and k8s distros. The name is similar to the `[Conformance]` cluster tests that
+are part of the Kubernetes Conformance Tests, which serves as the base for the
+Kubernetes certification program. The similarity in the name may be confusing,
+but this confusion will not be addressed in this KEP.
+
+One difference proposed in this document is to allow NodeConformance to be
+applied to `[Beta]` Features. Since beta features are enabled out of the box and
+in most deployments all Beta features stay enabled, applying Feature label to
+these tests may be misleading. This difference is intended, as we want to make
+sure Beta features as enabled out of the box are tested on PR validation and
+across different environments. This will give a better signal for GA-ing the
+feature.
+
+Proposed definition of `NodeConformance`:
+
+* `[NodeConformance]`: Node-level tests that validating behavior that doesn't
+  depend on specific Node capabilities being present, hardware, or feature set
+  of a dependency (like a container runtime), must be labeled as `[NodeConformance]`.
+  For the ease of test querying, each node-level test that is not testing alpha feature
+  (marked as `[FeatureGate:Foo][Alpha]`) is supposed to be either `NodeConformance`
+  or `Feature`.
+
+### See also
+
+- https://github.com/kubernetes/kubernetes/issues/59001
+- https://docs.google.com/document/d/1BdNVUGtYO6NDx10x_fueRh_DLT-SVdlPC_SsXjYCHOE/edit?usp=sharing
+- https://github.com/kubernetes/community/blob/32a1c14d04ff78684d78b827ac7c49f70352d509/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests
+- https://groups.google.com/g/kubernetes-sig-testing/c/8NVCt9UoV_Q/m/Vqz3_TidAwAJ?utm_medium=email&utm_source=footer
+
+### User Stories (Optional)
+
+#### Story 1
+
+1. New feature gate is introduced as Alpha.
+1. Tests for this functionality are marked as `[FeatureGate:Foo][Alpha]`.
+1. New test grid tab is added to run these tests while enabling the feature gate explicitly.
+1. Feature gate is promoted to Beta and enabled by default.
+1. Tests for this functionality are marked as `[FeatureGate:Foo][Beta]`.
```

Comment:
what is Alpha or Beta tag doing here? it seems that the NodeConformance tag is the discriminator